docs/user-guide/apps/llamacoder.md

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 102
+---
+
+# LlamaCoder 
+
+LlamaCoder is an open-source tool designed to generate small apps with a single prompt. It leverages LLM to help you quickly create and enhance React applications.
+
+## Prerequisites
+
+You will need a Gaia node ready to provide LLM services through a public URL. You can
+
+* [run your own node](../../node-guide/quick-start.md)
+* [use a public node](../nodes.md)
+
+In this tutorial, we will use a public Llama3 node.
+
+| Attribute | Value |
+|-----|--------|
+| API endpoint URL | https://llama.us.gaianet.network/v1 |
+| Model Name | llama |
+| API KEY | gaia |
+
+## Run the agent
+
+First, we will need to get the source code of the forked LlamaTutor
+
+```
+git clone https://github.com/second-state/llamacoder.git
+cd llamacoder
+```
+
+Next, configure the `.env` file.
+
+```
+cp .example.env .env
+```
+
+You will need to configure four parameters here.
```

Comment:
Three parameters. Also, you. An refer to the table above in case they need the values.

Reply:
Updated! Please check it again.